sos/plugins/openstack_novajoin.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2018 Red Hat, Inc., David Vallee Delisle <dvd@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin
+import os
+
+
+class OpenStackNovajoin(Plugin):
+    """OpenStack Novajoin
+    """
+    plugin_name = "openstack_novajoin"
+    profiles = ('openstack', 'openstack_undercloud')
+
+    def setup(self):
+        if self.get_option("all_logs"):
+            self.add_copy_spec([
```

Comment:
So we seem to have dropped `/etc/novajoin/` in this revision. Was that intentional?

We could collapse this into a single call using something like:

```python
nova_dir = "/var/log/novajoin/"
if not self.get_option("all_logs"):
    nova_dir += "*.log"

self.add_copy_spec([
    nova_dir,
    "/etc/novajoin/"
])
```